src/locales/id.json

```diff
@@ -0,0 +1,220 @@
+			"e-hentai": {
+				"category": "Total Doujin di e-hentai*",
+				"satania": "(Yang salah satunya sangatlah lucu)"
+			},
+			"rule34": "Total gambar di rule34.xxx*",
+			"gelbooru": "Total gambar di gelbooru*"
+		},
+		"as-of": "*tertanggal <span id=\"last-updated\"></span>"
+	},
+	"the-others-are-trash": {
+		"large-text": "Karakter lain itu sampah, tanpa terkecuali.",
+		"sub-text": "Mereka memang imut, tapi jika kamu tahu faktanya, sebenarnya tidak.",
+		"the-others-are-trash": {
+			"gabriel": {
+				"name": "Gabriel",
+				"name-detail": "Tenma White<br /><span lang=\"ja\">天真=ガヴリール=ホワイト</span>",
+				"description": {
+					"argument-1": "<b>NEET malas yang 100% sulit disukai</b>",
+					"argument-2": "Beneran, dia tidak punya kualitas yang baik",
+					"argument-3": "Orang-orang menyukainya mungkin karena dia dapat dikaitkan dengan kehidupan mereka",
+					"argument-4": "<b>Dan dapat mengaitkan kehidupanmu dengan Gab merupakan hal yang menjijikan</b>",
+					"argument-5": "<b>Dia banyak digunakan sebagai fanservice murahan</b>, seperti panty-shots dan sudut pengambilan kamera yang dipertanyakan",
+					"argument-6": "<b>Dasarnya dia itu Umaru</b>"
+				}
+			},
+			"raphiel": {
+				"name": "Raphiel",
+				"name-detail": "Shiraha Ainsworth<br /><span lang=\"ja\">白羽=ラフィエル=エインズワース</span>",
+				"description": {
+					"argument-1": "<b>Seorang sadis yang hanya ingin melihat orang lain menderita</b>",
+					"argument-2": "Sulit juga untuk disukai",
+					"argument-3": "Kecuali kamu orang yang sadis, yang juga berarti kamu adalah orang yang jahat",
+					"argument-4": "<b>Kemungkinan memiliki fetish kaki</b> (dia meminta Satania menjilat kakinya untuk sebuah roti)",
+					"argument-5": "Penggemarnya adalah orang cabul DAN juga orang yang menjijikan",
+					"argument-6": "Dia takut sama kodok lol"
+				}
+			},
+			"vignette": {
+				"name": "Vignette",
+				"name-detail": "Tsukinose April<br /><span lang=\"ja\">月乃瀬=ヴィネット=エイプリル</span>",
+				"description": {
+					"argument-1": "<b>MEMBOSAAAAAAANKAAAAAAAN</b>",
+					"argument-2": "Iya, dia tidak pernah melakukan sesuatu yang menarik",
+					"argument-3": "Kek, dia ndak ngapa-ngapain",
+					"argument-4": "<b>Orang-orang cuma suka sama dia karena dia itu imut</b>",
+					"argument-5": "<b>Kemungkinan besar dia dibuat hanya sebagai bahan moe tambahan</b>",
+					"argument-6": "Tidak pernah MENCOBA untuk menjadi jahat, meskipun dia itu seorang IBLIS"
+				}
+			}
+		}
+	},
+	"fanarts": {
+		"source": "Sumber",
+		"large-text": "Fanart yang sangat amat&nbsp;kawaii!",
+		"sub-text": "Nih, lihat... kurang imut apa coba?<noscript>Slideshow butuh JavaScript.</noscript>"
+	},
+	"autism": {
+		"large-text": "Juga, dia itu tidak autis, berhenti berkata&nbsp;seperti&nbsp;itu",
+		"sub-text": "Orang autis sering mengalami kesulitan dalam berkomunikasi dan berdiam diri di sudut mereka setiap saat, sedangkan Satania sama sekali tidak seperti itu.<br /><b>Kemungkinan kamu lebih autis daripada dia!</b>",
```

Comment:
Sorry, I didn't know what `&nbsp;` was used for so I just leave it where they originally are. DX

now that I look at it again, yeah it is awkward. its a bit hard to arrange the words in Bahasa since its word are more limited and tend to be longer than English, I'll try my best to make it as good and as poetic as i can.

if you think you see something weird, please tell me about it, I'll fix it :D